change/@ni-nimble-components-e882872b-5836-48a7-83b6-370d8c95d411.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Any idea why Chromatic reported a diff? Is this known intermittency?

Reply:
I'm not sure. It may be some new intermittency. Weirdly, it seemed to only affect _some_ of the tooltip matrix cases, and only on Firefox.

Reply:
@fredvisser shot in the dark, but I wonder if it's related to #1078. I don't think there's any action now but it would be something to keep an eye on in PRs today.